js/MineSweeper.js

```diff
@@ -383,7 +385,7 @@ jQuery(function ($) {
                 }
             }
 
-            fisherYates(array);
+            do {fisherYates(array)} while(array[0]==1)
```

Comment:
Sorry I don't have a coding standards thing set up yet, but please follow AirBnb's JavaScript coding style guide for code bracket, white spacing, and positioning: https://github.com/airbnb/javascript.
Except for indents, you can use 4 spaces to indent instead of AirBnb's two spaces.